### PR TITLE
Add rewrap command to docs

### PR DESCRIPTION
**Pull request by Will-Sommers**
This adds the rewrap command to the user docs. As an aside, I had a hard time getting it recognized over wrap and had to switch the command to "switch".

### CODE REVIEW

File: docs/user/README.md [hunk 1]
```diff
@@ -430,6 +430,9 @@ eg:
 `box wrap blue air`
 Wraps the token containing letter 'a' with a blue hat in square brackets.
 
+The rewrap command can be used to swap a given pair of symbols with another
+- `"curly rewrap <TARGET>"`: wraps the target with curly brackets, replacing the previous paired delimiter
```
Comment (pokey) [inline docs/user/README.md:434]: I believe the [default](https://github.com/cursorless-dev/cursorless-vscode/blob/b9355460a8e602bf47ec0b5ff1e609ad2f9017b3/cursorless-talon/src/actions/actions.py#L89) is

```suggestion
- `"curly repack <TARGET>"`: wraps the target with curly brackets, replacing the previous paired delimiter
```

Comment (pokey) [reply docs/user/README.md:434]: I don't feel strongly about the heading. The internal identifier is rewrap so maybe let's keep that for the heading? Cc/ @AndreasArvidsson

Comment (AndreasArvidsson) [reply docs/user/README.md:434]: Agreed, the  identifiers is what I prefer to use since everybody has different spoken forms.

**Review by pokey — APPROVED** [review]
Looks good!

**Pull request merged**